### PR TITLE
Develop update with documentation

### PR DESCRIPTION
**Pull request by vsecades**
Documentation update to move lib to release status.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,19 +90,118 @@ The load operation may take up to a few of seconds. Load Ver-ID using the `load`
...
+        }
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
+
+## Face detection session without asking for poses
```
Comment (jakubdolejs) [inline README.md:156]: The sample code in this section is identical to the code in the previous section.

File: README.md [hunk 2]
```diff
@@ -90,19 +90,118 @@ The load operation may take up to a few of seconds. Load Ver-ID using the `load`
...
+        return value;
+      });
+    } else {
+      throw verIdNotInitialized;
+    }
+  }
+~~~
 
 ## Session Response Format
+The callback of a successful session will contain [an object](https://appliedrecognition.github.io/Ver-ID-Person-Cordova-Plugin/classes/_ver_id_.sessionresult.html) that represents the result of the session.
```
Comment (jakubdolejs) [inline README.md:176]: The link refers to Cordova.

File: README.md [hunk 3]
```diff
@@ -90,19 +90,118 @@ The load operation may take up to a few of seconds. Load Ver-ID using the `load`
...
+    }
+  }
+~~~
+
 ## Project Samples
+Project samples can be found on the samples branch of this repository, to be used as a reference for development purposes.
 
 ## Module API Reference
-
+- [Ver-ID](https://appliedrecognition.github.io/Ver-ID-Person-Cordova-Plugin/modules/_ver_id_.html)
```
Comment (jakubdolejs) [inline README.md:206]: Cordova link

**Review by jakubdolejs — CHANGES_REQUESTED** [review]
Please see comments. Some of the links refer to the Cordova plugin and a couple of sections of the code are identical.

File: README.md (hunk 3 above)
Comment (vsecades) [reply README.md:210]: @jakubdolejs we'll need to move this to be serviced from https://github.com/AppliedRecognition/Ver-ID-Flutter/blob/develop/doc/api/index.html later on, or creating a Github page to load directly.

**Review by jakubdolejs — APPROVED** [review]

**Pull request merged**